phpBB/phpbb/console/command/db/list_command.php

```diff
@@ -39,6 +39,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
 		foreach ($this->load_migrations() as $name)
 		{
+			// Ignore non-migration files
+			if (\phpbb\db\migrator::is_migration($name) === false)
```

Comment:
It would also run twice when running the `migrate` cli command. Once inside load_migrations, then again inside the migration class